python/mdvtools/llm/code_manipulation.py

```
@@ -1,8 +1,14 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Optional
 import pandas as pd
 import regex as re
 from .templates import packages_functions
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)